### ISSUE TITLE
[Elixir] Implement new Concept Exercise: `erlang-libraries`

### ISSUE THREAD

**Issue by angelikatyborska** [exercism/v3#2306]
## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student that Erlang libraries are available from Elixir.

## Learning objectives

- Know how Elixir was written on top of Erlang, know what the BEAM is
- How to call Erlang functions
- Where to find the documentation for Erlang functions
- Know that when an Elixir library doesn't exist, the user should look for an Erlang library instead (e.g. no math operations in Elixir - `:math`, no C-style string formatting in Elixir - `:io_lib.format`)

## Out of scope

- ETS - too complicated, it would need its own exercise

## Concepts

- `erlang-libraries`

## Prerequisites

- `basics`
- `atoms`
- `charlists` if `:io_lib.format` is mentioned
- other, depending on the story

## Resources to refer to

- https://elixir-lang.org/getting-started/erlang-libraries.html
- https://inquisitivedeveloper.com/lwm-elixir-55/

## Analyzer

The analyzer should check that indeed Erlang functions are used instead of reimplementing them in Elixir.

## Story

Pick one or two simpler modules from the Erlang standard library (`timer`, `math`, `queue`, `rand`...) and try to come up with a simple story. Look through existing stories (`/reference/stories/`) to find inspiration or an existing story that can work for this exercise.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by angelikatyborska** [comment]
I think I finally have an idea for a story. I'll take a stab at this exercise.
